### ISSUE TITLE
HTML validator error for comment form

### ISSUE THREAD

**Issue by christophherr** [Getbeans/Beans#235]
`Error: Element legend not allowed as child of element p in this context. `

```[html]
<p class="uk-margin-top" data-markup-id="beans_comment_form[_comment]">
<legend  data-markup-id="beans_comment_form_legend[_comment]">
<!-- open output: beans_comment_form_legend_text[_comment] -->Comment *<!-- close output: beans_comment_form_legend_text[_comment] --></legend>
<textarea id="comment" class="uk-width-1-1" name="comment" required="" rows="8" data-markup-id="beans_comment_form_field[_comment]"></textarea>
</p>
```

Is the `<p>` needed for UIKit or can we change to `<fieldset>`?
It doesn't seem to cause a visual regression...
![fieldset](https://user-images.githubusercontent.com/8683126/40272960-72a1612e-5b85-11e8-9fcd-3aeb11fe63d8.jpg)

**Comment by JeffCleverley** [comment]
Has this error been fixed?

As far as I can see in the code it is:

```
<fieldset>
    <legend></legend>
    <textarea></textarea>
</fieldset>
```

Screenshot matches yours

![screen shot 2018-06-09 at 12 07 34 pm](https://user-images.githubusercontent.com/25419360/41190850-c4d98ad0-6bdd-11e8-9da5-7083408db4f3.png)

In php storm

<img width="1680" alt="screen shot 2018-06-09 at 12 08 20 pm" src="https://user-images.githubusercontent.com/25419360/41190855-d7d88a32-6bdd-11e8-83cf-2fb8e5f8fee4.png">

I will reach out on Slack Christoph, check to see what needs doing.

Jeff

**Comment by christophherr** [comment]
Hey @JeffCleverley,
sorry about that. 
Apparently, I kept the change after taking the screenshot in the file when I opened this [pull request]( 
https://github.com/Getbeans/Beans/pull/234/files#diff-7567fdb3fce72f197dfe8adacc701826R447) 

Fixed in #234.
